### DISCUSSION TITLE
RAM usage

### DISCUSSION THREAD

**Discussion by ymguerra** [fabiocaccamo/django-treenode#116] [General]
Hello, I was investigating a bit about tree structures and I found this library. In a personal project I am working on I have something similar to what this library help to solve. So I was testing this library and was working working great until I decide to added it to an existent model, this model has ~ 400mil records and then in the `post_migrate` signal when calling `sender_model.update_tree()` it use more than 20GB RAM.

Is there a work around for this?

**Comment by fabiocaccamo** [comment]
@ymguerra have you did this? https://github.com/fabiocaccamo/django-treenode#bulk-operations
